common/src/main/java/tachyon/worker/WorkerClient.java

```diff
@@ -78,18 +78,21 @@
   private final ClientMetrics mClientMetrics;
 
   /**
-   * Create a WorkerClient, with a given MasterClient.
+   * Creates a WorkerClient, with a given MasterClientBase.
```

Comment:
The constructor no longer takes a master client base + the parameters should have a description